resolwe_bio/processes/chip_seq/macs2.yml

```diff
@@ -93,13 +123,14 @@
             same coordination and the same strand. The 'auto' option makes MACS calculate the
             maximum tags at the exact same location based on binomal distribution using 1e-5 as
             pvalue cutoff and the 'all' option keeps all the tags. If an integer is given, at most
-            this number of tags will be kept at the same location.
+            this number of tags will be kept at the same location. The default is to keep one tag
+            at the same location.
         - name: qvalue
           label: Q-value cutoff
           type: basic:decimal
           required: false
-          disabled: "settings.pvalue"
-          disabled: "settings.pvalue_prepeak && case_prepeak"
+          disabled: "settings.pvalue && settings.pvalue_prepeak && tagalign"
+          hidden: "tagalign"
```

Comment:
Why not allow user to define `qvalue` when using pre-peakcall filtering?

Reply:
Fixed!